src/components/App.tsx

```diff
@@ -762,6 +764,27 @@ function App() {
     if (messages) navigator.clipboard.writeText(messages);
   };
 
+  /*//////////////////////////////////////////////////////////////
+                         RENAME NODE LOGIC
+  //////////////////////////////////////////////////////////////*/
+
+  const showRenameInput = () => {
+    takeSnapshot();
+
+    const selectedNode = nodes.find((node) => node.selected);
+    const nodeId = selectedNode?.id ?? selectedNodeId;
+
+    if (nodeId) {
+      setNodes((nodes) =>
+        modifyFluxNode(nodes, {
```

Comment:
could we rename this back to `modifyFluxNodeType`?

Reply:
I renamed this as I realised this function could be generalised to override any node option, not just types. But if you prefer I can revert to `modifyFluxNodeType`

Reply:
yeah i think i prefer the specific functions to signal intent rn, esp if we're not updating more then one property at a time